rplugin/python3/defx/view.py

```diff
@@ -37,13 +37,15 @@ def __init__(self, vim: Nvim,
             'silent keepalt edit', '[defx]')
 
         self._vim.current.window.options['list'] = False
+        self._vim.current.window.options['wrap'] = False
         self._options = self._vim.current.buffer.options
         self._options['buftype'] = 'nofile'
         self._options['swapfile'] = False
         self._options['modeline'] = False
         self._options['filetype'] = 'defx'
         self._options['modifiable'] = False
         self._options['modified'] = False
+        self._options['buflisted'] = False
```

Comment:
Sorry.  I don't want to disable the option.

Reply:
Ok. Can you just explain how it is useful? For me it leaves hanging buffer that i don't really need.